nix_update/__init__.py

```diff
@@ -28,9 +28,15 @@ def parse_args() -> Options:
         "--review", action="store_true", help="Run `nixpkgs-review wip`"
     )
     parser.add_argument("--format", action="store_true", help="Run `nixpkgs-fmt`")
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group()
```

Comment:
I don't think those options need to be exclusive. Sure, it does not make that much sense but it can logically be combined.